configs/xlml/pax/solutionsteam_pax_supported_config.py

```diff
@@ -69,13 +69,18 @@ def get_setup_cmds(
     raise RuntimeError(f"Please specify set up cmds for: {pax_version.value}.")
 
 
-def get_runtime_version(pax_version: PaxVersion) -> str:
-  if pax_version is PaxVersion.STABLE:
-    return vm_resource.RuntimeVersion.TPU_VM_V4_BASE.value
-  elif pax_version is PaxVersion.NIGHTLY:
-    return vm_resource.RuntimeVersion.TPU_UBUNTU2204_BASE.value
+def get_runtime_version(pax_version: PaxVersion, tpu_version: str) -> str:
+  if tpu_version == "5litepod":
```

Comment:
why are we using the 5litepod notation here? Shouldn't it be 5e?

Reply:
No, when creating the v5e, it's still v5litepod. The change starts from v5p.